back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -255,22 +255,6 @@ def destroy
 
   private
 
-  # Change counts on idea for values for phase, if filtered by phase
```

Comment:
Had already put this quickly onto the i1 branch but moved it so it would show up in this review